packages/gitbook/src/components/Header/HeaderLink.tsx

```diff
@@ -52,20 +65,20 @@ export async function HeaderLink(props: {
                 })();
                 return (
                     <Button
-                        href={target.href}
+                        href={target?.href}
```

Comment:
Can this result in a "dead button" since we don't pass an onClick? should we check for it?

Reply:
I simplified the whole logic @scazan

Reply:
looks great 👍